src/app/report/edit-display/edit-display.component.html

```diff
@@ -69,15 +69,26 @@
           ><i class="fa fa-save"></i></button>
         }
         <app-encoding-button [selectedNode]="selectedNode" (updatedMessageEvent)="editor.setNewReport($event)"></app-encoding-button>
-      }
-      @if (ReportUtil.isCheckPoint(selectedNode)) {
-        <div class="w-fit mx-1 my-0">
-          <select class="form-control" [(ngModel)]="stubStrategy" (ngModelChange)="openStubDifferenceModal($event)">
-            <option [ngValue]="-1" [selected]="selectedNode.stub === -1">Use report level stub strategy</option>
-            <option [ngValue]="0" [selected]="selectedNode.stub === 0">Always stub this checkpoint</option>
-            <option [ngValue]="1" [selected]="selectedNode.stub === 1">Never stub this checkpoint</option>
-          </select>
-        </div>
+
+        @if (selectedNode) {
+          @if (ReportUtil.isReport(selectedNode)) {
```

Comment:
ReportUtil.isReport should already return false if selectedNode is false, so this initial @if can be removed.

Reply:
You're right, changed it.